snips-nlu-ontology-parsers/src/builtin_entity_parser.rs

```diff
@@ -98,16 +102,22 @@ impl BuiltinEntityParser {
                 })
                 .collect()
         } else {
-            self._extract_entities(sentence, filter_entity_kinds)
+            self._extract_entities(sentence, reference_timestamp, filter_entity_kinds)
         }
     }
 
     fn _extract_entities(
         &self,
         sentence: &str,
+        reference_timestamp: Option<i64>,
         filter_entity_kinds: Option<&[BuiltinEntityKind]>,
     ) -> Vec<BuiltinEntity> {
-        let context = ResolverContext::default();
+        let reference_datetime: DateTime<Local> = Local.timestamp(
+            reference_timestamp.unwrap_or(Local::now().timestamp()), 0
+        );
+        let context = ResolverContext::new(
+            Interval::starting_at(Moment(reference_datetime), rustling_ontology::Grain::Second)
+        );
```

Comment:
I would rather for the following, in order to reduce the number of created objects and let rustling define the default behavior: 
```
let context = reference_timestamp
            .map(|ts|
                ResolverContext::new(
                    Interval::starting_at(Moment(Local.timestamp(ts, 0)), rustling_ontology::Grain::Second)))
            .unwrap_or_else(|| ResolverContext::default());
```

Reply:
Done